### PR TITLE
[TW-1392] migration guide swagger

### PR DESCRIPTION
**Pull request by petewitcher-postman**
Added a new doc on importing a Swagger API. Included a leftnav item for a separate doc from TW-1391 to avoid a merge conflict.

### CODE REVIEW

File: src/pages/docs/getting-started/importing-and-exporting/importing-from-swagger.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+## Import a Swagger API
+
+To import a Swagger API, do the following:
+
+1. Select **Import**.
+1. Select files, folders, enter a link to the API, or paste your raw text.
+
+    > You can also import an API definition from a code repository. Learn more about [importing and exporting](/docs/getting-started/importing-and-exporting/importing-and-exporting-overview/).
+
+1. Confirm the name, format, and what you would like your data to import as.
```
Comment (akinard-postman) [inline src/pages/docs/getting-started/importing-and-exporting/importing-from-swagger.md:38]: Is this correct? I was checking and the screen that followed during the process looked like this:
Screenshot 2023-09-29 at 9 05 25 AM

File: src/pages/docs/getting-started/importing-and-exporting/importing-from-swagger.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    name: "Blog posts"
+  - type: link
+    name: "Postman Essentials: Exploring the Collection Format"
+    url: "https://blog.postman.com/postman-essentials-exploring-the-collection-format/"
+  - type: link
+    name: "Differences between Postman Collections and the collection format"
+    url: "https://blog.postman.com/differences-between-postman-collections-and-collection-format/"
+---
+
+Postman can import APIs created with the Swagger toolset and any API that follows the OpenAPI specification. You can import your API as a standard collection (described below), or you can use the [Postman API Builder](/docs/designing-and-developing-your-api/the-api-workflow/) to import both the definition and a linked collection.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/getting-started/importing-and-exporting/importing-from-swagger.md:25]: Seems like [this section](https://learning.postman.com/docs/designing-and-developing-your-api/importing-an-api/#importing-api-definitions) is meant to explain the Postman API Builder instructions, but I'm wondering if it's outdated. Based on the text here and Ashley's screenshot below, it seems like the Postman API Builder instructions would be about selecting the second radio button, but I'm not 100% positive.

Comment (MatthewPinsker-postman) [inline src/pages/docs/getting-started/importing-and-exporting/importing-from-swagger.md:25]: Also should we call out specific OpenAPI versions that we can import into Postman? I think the versions are OpenAPI 3.0 and 3.1, and Swagger 1.2 and 2.0 based on [this section](https://learning.postman.com/docs/designing-and-developing-your-api/importing-an-api/#supported-api-definitions-formats). Although, I'm not 100% positive if this information is relevant to this area of the LC docs.

File: src/pages/docs/getting-started/importing-and-exporting/importing-from-swagger.md (hunk 1 above)
Comment (petewitcher-postman) [reply src/pages/docs/getting-started/importing-and-exporting/importing-from-swagger.md:38]: Thanks, changed to:

1. Choose to import your API as a **Postman Collection** or as **OpenAPI 3.0 with a Postman Collection**.

File: src/pages/docs/getting-started/importing-and-exporting/importing-from-swagger.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (petewitcher-postman) [reply src/pages/docs/getting-started/importing-and-exporting/importing-from-swagger.md:25]: Done, thanks.

Comment (petewitcher-postman) [reply src/pages/docs/getting-started/importing-and-exporting/importing-from-swagger.md:25]: Added a link to the supported versions.

File: src/pages/docs/getting-started/importing-and-exporting/importing-from-swagger.md (hunk 1 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/getting-started/importing-and-exporting/importing-from-swagger.md:38]: Is the text in the second radio button dynamic based on the OpenAPI version? I tested uploading API definitions in versions 3.0 and 2.0. Looks like the version number in the radio button text changes accordingly. Although, it looks like we also only mention "3.0" on [this page](https://learning.postman.com/docs/getting-started/importing-and-exporting/importing-from-git/).

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM! Added questions about a few steps.

**Review by akinard-postman — APPROVED** [review]

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**